client_source/sqlplus/ut_run.sql

```diff
@@ -70,7 +85,12 @@ Parameters:
 
 whenever sqlerror exit failure
 whenever oserror exit failure
-conn &1
+
+define client_path="&1"
```

Comment:
accordint to the invokation example in the beginning of the file (`ut_run.sql user/password@database [-p=(ut_path|ut_paths)] [-c] [-f=format [-o=output] [-s] ...]` 
&1 corresponds to user/password@database and all the others are wrong...